src/components/VTranslationStatusBanner.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <VNotificationBanner v-show="!shouldHideBanner" @close="dismissBanner">
+  <VNotificationBanner
+    v-if="needsTranslationBanner"
+    :id="bannerKey"
+    variant="informational"
+  >
     {{
       // eslint-disable-next-line @intlify/vue-i18n/no-raw-text
       '⚠️'
```

Comment:
We could add this symbol to ignored strings that @krysal created.

Reply:
Should we use that list for _anything_ you think? The ones Krystle added made sense to me because they're used in a couple of places in the app, I think... Hopefully raw emoji aren't that common in the app :thinking:

Reply:
Now that you say so, I guess it does make sense to use that list for repeated strings, and not for single emojis :)